docs/readthedocs/models/input/grid/line.rst

```diff
@@ -59,11 +59,6 @@ Entity Model
 | geoPosition       | --   | | Line string of geographical locations describing the |
 |                   |      | | position of the line                                 |
 +-------------------+------+--------------------------------------------------------+
-| olmCharacteristic | --   | | Characteristic of possible overhead line monitoring  |
```

Comment:
Within the line model, we need some reference to the OLM model. That is, that I'd like to keep this field and possibly rename it to `olmModel` or so. Moreover, it would also be desired, that we can have simple models as well. That said, would you mind proposing a second olm model (in the way with the above file) to keep the functionality of this simple model?